### PR TITLE
feat: add nutanix

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nutanix`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes
| Tickets          | Fix #MANAGER-7450
| License          | BSD 3-Clause

## Description

Add nutanix module

### CODE REVIEW

File: packages/manager/apps/nutanix/src/app.module.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          detachPreloader();
+          $rootScope.$broadcast('app:started');
+          unregisterHook();
+        });
+      },
+    )
+    .run(
+      /* @ngInject */ ($state) => {
+        $state.defaultErrorHandler((error) => {
+          console.log(error);
```
Comment (radireddy) [inline packages/manager/apps/nutanix/src/app.module.js:52]:
```suggestion
   
```

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/nutanix/src/dashboard/controller.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export default class NutanixCtrl {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    console.log('Hello', this.$translate.instant('world'));
```
Comment (mohammed-zahaf) [inline packages/manager/modules/nutanix/src/dashboard/controller.js:8]: must we kept that console @radireddy  ?

File: packages/manager/modules/nutanix/src/onboarding/constants.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const GUIDES = [];
```
Comment (mohammed-zahaf) [inline packages/manager/modules/nutanix/src/onboarding/constants.js:1]: Empty ?

Comment (radireddy) [reply packages/manager/modules/nutanix/src/onboarding/constants.js:1]: this is auto generated code, some data will be added as we continue to work on this project.

File: packages/manager/modules/nutanix/src/dashboard/controller.js (hunk 2 above)
Comment (radireddy) [reply packages/manager/modules/nutanix/src/dashboard/controller.js:8]: this is auto generated framework code, all these will be deleted or taken care during implementation.

**Review by ganeshkumar1989 — APPROVED** [review]

**Pull request merged**